src/main/java/org/broadinstitute/hellbender/tools/funcotator/Funcotator.java

```diff
@@ -427,6 +457,35 @@ public void closeTool() {
 
     //==================================================================================================================
 
+    /**
```

Comment:
Shouldn't this be in the superclass?  Or the Argument sub-system?

Reply:
This seems like useful functionality to automate and put in every command line....

Reply:
That's a good point.  

@LeeTL1220 to be clear, are you saying it should be generic and accept any argument for a tool as part of the config file?  That is, you can give the config file arg and the config file can contain all args for the tool?  

I think making it completely generic would be difficult because of the way `Owner` is built (but not impossible).

@droazen care to weigh in?

Reply:
@jonn-smith That is what I meant.

Reply:
In barclay, there is a special argument to load the arguments from a file (`--arguments_file`, see [barclay#SpecialArgumentsCollection](https://github.com/broadinstitute/barclay/blob/master/src/main/java/org/broadinstitute/barclay/argparser/SpecialArgumentsCollection.java)). This is included in `CommandLineProgram`, so it should be doing that job. I think that configuration files makes more sense for concrete arguments/tools that are system-wide.

In the case of Funcotator makes sense because the config file can be provided with the sources and ask the user to modify them instead of provide single arguments, but it can also be an `arguments_file` instead.